### ISSUE TITLE
db:dump - "...prefix pass instead of password is deprecated..."

### ISSUE THREAD

**Issue by robbieaverill** [netz98/n98-magerun#749]
Needs a tweak i.e. `--pass` -> `--password`

Version: 1.97.7

```
$ n98-magerun.phar db:export
# Enter some filename
Start dumping database magento_db_name to file some_filename.sql
Warning: Using unique option prefix pass instead of password is deprecated and will be removed in a future release. Please use the full name instead.
Finished
```

**Comment by robbieaverill** [comment]
I can't see any reference to it - hmmm...

**Comment by robbieaverill** [comment]
Ah - fixed in 8be2b18 - closing :)
